packages/storage/src/tasksnapshot.ts

```diff
@@ -19,13 +19,15 @@ import { Metadata } from './metadata';
 import { Reference } from './reference';
 import { UploadTask } from './task';
 
-export class UploadTaskSnapshot {
-  constructor(
-    readonly bytesTransferred: number,
-    readonly totalBytes: number,
-    readonly state: TaskState,
-    readonly metadata: Metadata,
-    readonly task: UploadTask,
-    readonly ref: Reference
-  ) {}
+export interface UploadTaskResumableSnapshot {
+  readonly bytesTransferred: number;
+  readonly totalBytes: number;
+  readonly state: TaskState;
+  readonly metadata: Metadata;
+  readonly task: UploadTask;
+  readonly ref: Reference;
+}
+export interface UploadTaskSnapshot {
```

Comment:
Missing space. Should we also flip the declaration order?

Reply:
Done, makes sense.